### ISSUE TITLE
Uncaught TypeError: Cannot read property 'props' of null

### ISSUE THREAD

**Issue by impervius** [khanhas/spicetify-cli#638]
![image](https://user-images.githubusercontent.com/63152795/119024880-3d5dc100-b9ac-11eb-8569-a8286ad18da0.png)
```
Uncaught TypeError: Cannot read property 'props' of null
    at Object._addItems (spicetifyWrapper.js:759)
    at MutationObserver.<anonymous> (vendor~xpui.js:20)
```
This is with no extensions turned on, with freshly installed `spotify` and `spicetify-cli` from the AUR.
```
[Patch]

[Setting]
spotify_path            = /opt/spotify
spotify_launch_flags    =
check_spicetify_upgrade = 0
prefs_path              = /home/anon/.config/spotify/prefs
current_theme           = SpicetifyDefault
color_scheme            =
inject_css              = 1
replace_colors          = 1
overwrite_assets        = 0

[Preprocesses]
disable_sentry        = 1
disable_ui_logging    = 1
remove_rtl_rule       = 1
expose_apis           = 1
disable_upgrade_check = 1

[AdditionalOptions]
extensions  =
custom_apps =

; DO NOT CHANGE!
[Backup]
version = 1.1.56.595.g2d2da0de

```

```
➜ spicetify --version
2.2.2
 

➜ spotify --version
spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by spotify)
Spotify version 1.1.56.595.g2d2da0de, Copyright (c) 2021, Spotify Ltd
```

I can't seem to use any extension: I tried turning on `shuffle+.js` and `fullAppDispaly.js` and they didn't work, so I looked in the console and saw these errors. I then removed the extensions (and ran `spicetify update` and `spicetify apply`) and the errors still came up. Then I completley reinstalled spotify and spicetify and removed `~.config/spicetify` and the issue persists.

**Comment by khanhas** [comment]
https://github.com/khanhas/spicetify-cli/issues/637#issuecomment-845583999

**Comment by khanhas** [comment]
Should be fixed now in spicetify v2.2.3
